jekyll/_cci2/language-ruby.md

```diff
@@ -7,60 +7,58 @@ categories: [language-guides]
 order: 8
 ---
 
-*[Tutorials & 2.0 Demo Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Ruby*
+*[Tutorials & 2.0 Sample Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Ruby*
 
 This guide will help you get started with a Ruby on Rails application on CircleCI. 
 
 ## Overview
 
 If you’re in a rush, just copy the sample configuration below into a `.circleci/config.yml` in your project’s root directory and start building.
 
-Otherwise, we recommend reading our [walkthrough](#config-walkthrough) for a detailed explanation of our configuration.
-
-We maintain a reference Ruby on Rails project to show how to build Ruby on CircleCI 2.0:
+CircleCI maintains the sample Ruby on Rails project at the following links:
```

Comment:
The project exists and is maintained at only one of the links (the GitHub repository). The CircleCI link is also useful, but it is a link to a CircleCI project, rather than the Rails example.

Reply:
It is the Ruby project build page.

Reply:
That's true, but code is only changed in the GitHub repository. Nothing is being _maintained_ at the Ruby project build page.